### PR TITLE
fix github action for publishing stubs

### PR DESCRIPTION
**Pull request by madhur-ob**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -29,13 +29,13 @@ jobs:
       run: |
         cd ./stubs && python3 setup.py sdist bdist_wheel --universal && cd -
     - name: Publish metaflow-stubs package
-      uses: pypa/gh-action-pypi-publish@9b8e7336db3f96a2939a3e9fa827c62f466ca60d # master
```
Comment (romain-intel) [inline .github/workflows/publish.yml:32]: - I think we just need to do release/v1 according to the doc

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -29,13 +29,13 @@ jobs:
...
+      uses: pypa/gh-action-pypi-publish@release/v1.8
       with:
         user: __token__
         password: ${{ secrets.pypi_password }}
-        packages-dir: ./stubs/dist
```
Comment (romain-intel) [inline .github/workflows/publish.yml:36]: the doc does say `packages-dir`. I think it was "fixed": https://github.com/pypa/gh-action-pypi-publish/blob/c12cc61414480c03e10ea76e2a0a1a17d6c764e2/action.yml#L28

File: .github/workflows/publish.yml (hunk 1 above)
Comment (madhur-ob) [reply .github/workflows/publish.yml:32]: using hash for `v1.8.11`

File: .github/workflows/publish.yml (hunk 2 above)
Comment (madhur-ob) [reply .github/workflows/publish.yml:36]: reverted...

**Pull request merged**